content/docs/user-guide/visualizing-plots.md

```diff
@@ -276,6 +276,8 @@ The `linear` template is the default. It can be changed with the `--template`
 (`-t`) option of `dvc plots show` and `dvc plots diff`. The argument provided to
 `--template` can be a (built-in) template name or a path to a [custom template].
 
+[custom templates]: /doc/command-reference/plots/templates
+
```

Comment:
It's already in line 299

```suggestion
```

But I just noticed that line 307 should reuse the ref link (currently it repeats the whole path).

Reply:
Actually, the ref above should be `custom template` (without the `s`) and is broken now. Not sure if there's a way to consolidate here if they all need different text to show. Otherwise I can just fix the link above.

Reply:
Yes we should consolidate...